mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -443,6 +443,11 @@ func awsS3DataSchema() *schema.Resource {
 				Type:        schema.TypeString,
 				Description: `S3 Bucket name.`,
 			},
+			"path": {
+				Required:    true,
```

Comment:
This new field should not be required. Otherwise, this change will be a breaking change.

Reply:
definitely! I fixed

https://github.com/GoogleCloudPlatform/magic-modules/pull/7932/commits/757076dd094eaed7ef02b2f5714dc420d473d39a